app/build.gradle.kts

```diff
@@ -112,38 +114,32 @@ dependencies {
     implementation(project(":lib-xml"))
     implementation(project(":theme-innovation-cup"))
 
-    implementation(Dependencies.composeMaterial3)
-    implementation(Dependencies.coreKtx)
-    implementation(Dependencies.kotlinReflect)
+    implementation(Dependencies.accompanistFlowLayout)
+    implementation(Dependencies.accompanistPager)
+    implementation(Dependencies.accompanistPagerIndicators)
+    implementation(Dependencies.accompanistSystemUiController)
+    implementation(Dependencies.activityCompose)
     implementation(Dependencies.appCompat)
-    implementation(Dependencies.material)
+    implementation(Dependencies.browser)
+    implementation(Dependencies.coil)
+    implementation(Dependencies.coilCompose)
+    implementation(Dependencies.composeMaterial3)
     implementation(Dependencies.composeUi)
-    implementation(Dependencies.lifecycleViewModelKtx)
+    implementation(Dependencies.composeUiTooling)
```

Comment:
This was in `debugImplementation` previously, is it correct?

Reply:
Exactly, and I don't know why 🤷